### PR TITLE
Add new licenses

### PR DESCRIPTION
**Pull request by AyanSinhaMahapatra**
These were added by @DennisClark

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁
* [ ] Updated documentation pages (if applicable)
* [ ] Updated CHANGELOG.rst (if applicable)

### CODE REVIEW

File: src/licensedcode/data/licenses/rh-eula-lgpl.LICENSE [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
 key: rh-eula-lgpl
-short_name: Red Hat EULA JBoss Enterprise Middleware License
-name: Red Hat EULA JBoss Enterprise Middleware License
+short_name: Red Hat LGPL-Based EULA
+name: Red Hat LGPL-Based End User License Agreement
 category: Copyleft Limited
 owner: Red Hat
-homepage_url: https://www.redhat.com/f/pdf/licenses/GLOBAL_EULA_JBoss_English_20101110.pdf
+homepage_url: https://www.redhat.com/en/about/eulas
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/rh-eula-lgpl.LICENSE:7]: This lgpl EULA does not look the same as JBoss one:
- https://www.redhat.com/licenses/Red_Hat_LGPL-Based_EULA_20191118.pdf
- https://www.redhat.com/f/pdf/licenses/GLOBAL_EULA_JBoss_English_20101110.pdf

these are super similar but are they really the same? We want to keep both text URLs in all cases and we may need some rules?

File: src/licensedcode/data/licenses/rsalv2.LICENSE [hunk 2]
```diff
@@ -6,7 +6,7 @@ category: Source-available
 owner: Redis
 homepage_url: https://redis.com/legal/rsalv2-agreement/
 spdx_license_key: LicenseRef-scancode-rsalv2
-faq_url: https://redis.com/blog/rsalv2-sspl-announcement/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/rsalv2.LICENSE:9]: Keep this as an other URL

File: src/licensedcode/data/licenses/rh-eula.LICENSE [hunk 3]
```diff
@@ -4,7 +4,7 @@ short_name: Red Hat EULA
 name: Red Hat EULA for Enterprise Linux and Applications
 category: Copyleft
 owner: Red Hat
-homepage_url: http://www.redhat.com/en/about/red-hat-end-user-license-agreements
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/rh-eula.LICENSE:7]: Keep this under other_urls

File: src/licensedcode/data/licenses/saxix-mit.LICENSE [hunk 4]
```diff
@@ -2,12 +2,12 @@
 key: saxix-mit
 short_name: Stefano Apostolico MIT License
 name: Stefano Apostolico MIT License
-category: Permissive
+category: Proprietary Free
 owner: Stefano Apostolico
 homepage_url: https://github.com/saxix/django-adminactions/blob/develop/LICENSE
+notes: this is the MIT license text with an extra clause wrt. commercial usage notification.
 spdx_license_key: LicenseRef-scancode-saxix-mit
 minimum_coverage: 95
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/saxix-mit.LICENSE]: I would move this to 100 and create a rule with these lines https://github.com/nexB/scancode-toolkit/pull/3778/files#diff-fba6a45f6fe77d22e39582632b7061815e79911610fe43ec50028bb8a64a8428R15-R33 (minus the MIT License line) and make the extra clause a key phrase.

**Review by pombredanne — APPROVED** [review]
Looking good. A few minor nits that are not essential

**Comment by AyanSinhaMahapatra** [comment]
Merged, thanks a lot for reporting these @LeChasseur @armijnhemel, @pabs3 and @pombredanne  and thanks for adding these @DennisClark

**Pull request merged**